pkg/controller/virtualmachine/virtualmachine_controller.go

```diff
@@ -114,6 +114,10 @@ func (r *ReconcilePolicy) addFinalizerAndUpdate(virtualMachine *kubevirt.Virtual
 		return nil
 	}
 
+	if err := r.poolManager.MarkVMAsReady(virtualMachine); err != nil {
```

Comment:
For the commit message. Please explain when does it happen and what are the consequences.